### PR TITLE
feat: Adds `colspan` support to `th` and `td` elements

### PR DESCRIPTION
**Pull request by esoubihe**

### CODE REVIEW

File: src/Table/index.js [hunk 1]
```diff
@@ -25,10 +24,11 @@ Table.Body = ({ children, className }) => (
   </tbody>
 );
 
-Table.HeaderCell = ({ children, className }) => (
+Table.HeaderCell = ({ children, className, colSpan }) => (
   <th
     tw="px-6 py-2 bg-white text-left text-xs leading-5 font-medium text-foreground uppercase tracking-wider"
     className={className}
+    colspan={colSpan}
```
Comment (maxweldsouza) [inline src/Table/index.js:31]:
```suggestion
    colSpan={colSpan}
```

Attributes need to be camelCase. Although this works, it throws a warning.

File: src/Table/index.js [hunk 2]
```diff
@@ -48,8 +48,8 @@ Table.Row = ({ children, className, onClick, isSelectable }) => (
   </tr>
 );
 
-Table.Cell = ({ children, className }) => (
-  <td tw="px-6 py-4" className={className}>
+Table.Cell = ({ children, className, colSpan }) => (
+  <td tw="px-6 py-4" className={className} colspan={colSpan}>
```
Comment (maxweldsouza) [inline src/Table/index.js:52]:
```suggestion
  <td tw="px-6 py-4" className={className} colSpan={colSpan}>
```

**Review by maxweldsouza — COMMENTED** [review]
Might be better to pass all remaining props to the elements.

**Comment by esoubihe** [comment]
> Might be better to pass all remaining props to the elements.

What do you mean?

**Comment by maxweldsouza** [comment]
These elements have a few more attributes like rowSpan, abbr, as well as aria-* and data-* attributes. We should use {...props} so that there is support for all such cases.

**Comment by esoubihe** [comment]
> These elements have a few more attributes like rowSpan, abbr, as well as aria-* and data-* attributes. We should use {...props} so that there is support for all such cases.

So we might just remove `colSpan` and provide props for everything instead?

**Comment by esoubihe** [comment]
> Yes. We can even remove className. We should only destructure props that we need to modify or handle in some way. Everything else can pass through. There are too many attributes to be able to list manually. https://use-form.netlify.app/interfaces/_node_modules__types_react_index_d_.react.thhtmlattributes.html

Thanks, Max, this is very helpful.

**Review by maxweldsouza — APPROVED** [review]
The dist file is still the old one. That can be updated during release.

**Pull request merged**